tools/updateIonRangeSlider.R

```diff
@@ -1,35 +1,57 @@
 #!/usr/bin/env Rscript
-
-# This script copies resources from ion.RangeSlider to shiny's inst
-# directory. The ion.rangeSlider/ project directory should be on the same level
-# as the shiny/ project directory.
+# Retrieves a particular version of ion.rangeSlider:
+#  https://github.com/IonDen/ion.rangeSlider
+# Then applies patches from tools/ion.rangeSlider-patches.
 
 # The version of ion.rangeSlider that we use is modified from the base version
 # in two ways:
 # * In our version, mouse events on the slider are not propagated to lower
 #   layers (#711, #1630).
 # * We include a custom skin for Shiny.
+# * When .update() is called, the `.removeProp("tabindex")` is wrapped in
+#   try-catch. This is because this line results in an error with jQuery 3 and
+#   PhantomJS. (#2587)
```

Comment:
Minor nitpick: I think I'd prefer if these bullets appeared where the patches are applied, and have them reference the patch filenames